### DISCUSSION TITLE
Password protect Ant media Application Pages.

### DISCUSSION THREAD

**Discussion by USAMAWIZARD** [ant-media/Ant-Media-Server#5158] [General]
is there any way to password protect all the applications pages in ams like LiveApp and WebRTCAppEE.
user should not be able to visit this sample pages without a password.

**Comment by yashtandon113** [comment]
@USAMAWIZARD 
Thanks for your question.

You can quickly make the page password protected using simple HTML code. You just need to add below code to the body part on sample HTML pages. You can define your password in a password field.

```
 <script>
         var password = "testing123";
         (function passcodeprotect() {
            var passcode = prompt("Enter PassCode");
            while (passcode !== password) {
               alert("Incorrect PassCode");
               return passcodeprotect();
            }
         }());
         alert('Welcome');
 </script>
```
You can make the changes in this code as per your requirements.

I hope it helps.

**Reply by mekya** [reply]
This is just a front-end stuff and the password is on the page.  If I want to protect the page, this will not be the way I prefer.
 
Could you please provide a better solution that is more secure?

**Reply by yashtandon113** [reply]
@mekya 
Thanks for your feedback.
Sure, I will look into it and try to provide more secure solution.
